doc/host_config_schema/cchost_config.json

```diff
@@ -202,7 +202,7 @@
                   "retry_timeout": {
                     "type": "string",
                     "default": "1000ms",
-                    "description": "Interval (time string) at which the node sends join requests to the existing network"
+                    "description": "Interval (time string) at which the node sends join requests to the existing service. This should be less than the value of 'consensus.election_timeout' set on the primary node of the existing service to join."
```

Comment:
This is just documentation to avoid the primary node detecting new joiners as inactive in between the time they have been trusted by the consortium and the time they actually join the network ([more details here](https://microsoft.github.io/CCF/main/architecture/consensus/1tx-reconfig.html)). The default values, 5s for the election timeout and 1s for the join retry, are sensible.